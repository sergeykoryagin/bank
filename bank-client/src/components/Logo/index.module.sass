@import "src/styles/mixins"

.logo
    @include font-header-2
    color: $color-blue-300