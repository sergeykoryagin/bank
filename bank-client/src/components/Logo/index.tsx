import { VFC } from 'react';
import cn from 'classnames';
import styles from './index.module.sass';

interface Props {
    className?: string;
}

export const Logo: VFC<Props> = ({ className }: Props) => {
    return <span className={cn(styles.logo, className)}>Bankomat Online</span>;
};
