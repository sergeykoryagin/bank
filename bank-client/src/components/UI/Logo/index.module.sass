@import "../../../styles/mixins"

.logo
    @include font-h1
    color: $color-blue-300
    text-align: center