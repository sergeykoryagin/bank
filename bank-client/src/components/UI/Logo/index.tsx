import { VFC } from 'react';
import cn from 'classnames';
import styles from 'components/UI/Logo/index.module.sass';

interface Props {
    className?: string;
}

export const Logo: VFC<Props> = ({ className }: Props) => {
    return (
        <span className={cn(styles.logo, className)}>
            Bankomat
            <br />
            Online
        </span>
    );
};
