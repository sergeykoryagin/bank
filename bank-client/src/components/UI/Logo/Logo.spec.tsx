import { render, screen } from '@testing-library/react';
import { Logo } from 'components/UI/Logo/index';


describe('Logo component', () => {
    it('should renders correctly', () => {
        const {container} = render(<Logo />);
        expect(screen.getByText(/bankomat/i)).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    });


});
