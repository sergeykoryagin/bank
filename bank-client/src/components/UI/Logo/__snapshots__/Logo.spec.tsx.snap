// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should renders correctly 1`] = `
<div>
  <span
    class="logo"
  >
    Bankomat
    <br />
    Online
  </span>
</div>
`;
